HALO/smart.yaml

```diff
@@ -0,0 +1,11 @@
+plugins:
+  source:
+    - module: intake_xarray
+
```

Comment:
This catalog doesn't contain references to `opendap` datasets, so it can be read without the use of any plugins.

```suggestion
```